c/include/prima/prima.h

```diff
@@ -68,9 +68,7 @@ typedef enum {
     PRIMA_NULL_X0 = 112,
     PRIMA_NULL_RESULT = 113,
     PRIMA_NULL_FUNCTION = 114,
-    PRIMA_PROBLEM_SOLVER_MISMATCH_NONLINEAR_CONSTRAINTS = 115,
```

Comment:
I decided to move these to the Python code. The main reason is that we will need to change the behavior for SciPy since it allows functions to run with mismatched constraints. So since we will have different behavior in two packages it's not feasible to have the behavior here.

Reply:
> SciPy since it allows functions to run with mismatched constraints.

IMHO, this is a bug of SciPy, especially considering its return status when this happens.

Reply:
So what action would you suggest to take for this PR?